validator/python/copy.py

```diff
@@ -1,433 +0,0 @@
-"""Generic (shallow and deep) copying operations.
```

Comment:
Wait, we don't get to have our own custom implementation of `copy`? *sadface*.
Jokes aside, very nice cleanup ;)